dGame/dComponents/VehiclePhysicsComponent.cpp

```diff
@@ -93,7 +93,7 @@ void VehiclePhysicsComponent::Serialize(RakNet::BitStream* outBitStream, bool bI
 	}
 
 	if (bIsInitialUpdate) {
-		outBitStream->Write<uint8_t>(5);
+		outBitStream->Write<uint8_t>(GeneralUtils::GenerateRandomNumber<uint32_t>(0, 7));
```

Comment:
It seems a bit weird to have this kind of logic in the serialization. Would prefer if this was in the constructor, or better in the logic of the racing manager. This would also make sure that clients see the same player doing different animations because they loaded in at separate times.

Reply:
Ah yeah good catch.  Didn't think about that bit

Reply:
I think this would be better in the vehiclePhysicsComponent since the behavior is only attached to this component and adding more to the racing manager wouldn't be ideal since it is quite large as it is.  Just my 2 cents though